### PR TITLE
#230 Add case priority script example

### PR DESCRIPTION
**Pull request by hilton**
Adds a new nested case variable.

I’ll merge immediately, as this is already live.

### CODE REVIEW

**Comment by hilton** [comment]
> I’ll merge immediately, as this is already live.

On second thoughts, I’ll let you translate it, which should fix the spelling test failure.

File: docs/include_variables/case.rst [hunk 1]
```diff
@@ -41,6 +41,10 @@ An email has a composite type, with the following properties.
      - ``dueDate``
      - :ref:`type-date`
      - Optional editable due date
+   * - Priority
+     - ``priority``
+     - :ref:`type-text`
+     - Case priority - values ``'0'`` (high) to ``'3'`` (low)
```
Comment (TimKam) [inline docs/include_variables/case.rst:47]: @hilton  Shouldn't it be the other way around? 0 is low and 3 is high?

Comment (hilton) [reply docs/include_variables/case.rst:47]: Probably, but that isn't how it works in the app :(

Comment (TimKam) [reply docs/include_variables/case.rst:47]: Ok, thanks for clarifying

**Pull request merged**